unit_tests/test_get_non_system_ks_cf_list.py

```diff
@@ -1,134 +0,0 @@
-from __future__ import absolute_import
```

Comment:
where are the tests replace those test ?, or you think they are not needed ?

Reply:
To use the session I need a real cluster.
This entire file contained tests which are based on factual nodes